spec/controllers/admin/roles_controller_spec.rb

```diff
@@ -3,11 +3,16 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::Admin::RolesController, type: :controller do
-  before { request.headers['ACCEPT'] = 'application/json' }
+  let(:user) { create(:user, role: create(:role, name: 'User')) }
+
+  before do
+    request.headers['ACCEPT'] = 'application/json'
+    session[:user_id] = user.id
+  end
 
   describe 'roles#index' do
     it 'returns the list of roles' do
-      roles = [create(:role, name: 'Hokage'), create(:role, name: 'Jonin'), create(:role, name: 'Chunin'), create(:role, name: 'Genin')]
+      roles = [create(:role, name: 'Hokage'), create(:role, name: 'Jonin'), create(:role, name: 'Chunin'), user.role]
```

Comment:
For consistency only, it may be better to do:
```
roles = roles = [create(:role, name: 'Hokage'), create(:role, name: 'Jonin'), create(:role, name: 'Chunin')]
roles << user.role
```